python/cudf/cudf/tests/test_reshape.py

```diff
@@ -206,8 +208,15 @@ def test_df_stack_multiindex_column_axis(columns, index, level, dropna):
     )
     gdf = cudf.from_pandas(pdf)
 
-    got = gdf.stack(level=level, dropna=dropna)
-    expect = pdf.stack(level=level, dropna=dropna)
+    with pytest.warns(FutureWarning):
+        got = gdf.stack(level=level, dropna=dropna, future_stack=False)
+    with expect_warning_if(PANDAS_GE_220):
+        expect = pdf.stack(level=level, dropna=dropna, future_stack=False)
```

Comment:
Could we make these pandas calls compatible for when we eventually test with pandas=2.0?

Reply:
It would make this test logic a lot more complicated and hard to understand. Do you want to xskip this test for pandas<2.1.0?

Reply:
Ah sure we can do that instead. Good idea

Reply:
Done